README.md

```diff
@@ -119,6 +119,7 @@ To disable `@import` resolving by `css-loader` set the option to `false`
 
 ```css
 @import url('https://fonts.googleapis.com/css?family=Roboto');
+@import '~module/styles.css';
```

Comment:
I don't think this belongs here since the example is for **disabling** `@import` resolving and why/when one would want to do that :)

Reply:
Whoops! How does it look now?